.github/actions/rust-build-release/src/main.py

```
@@ -14,21 +14,39 @@
 import typing as typ
 from pathlib import Path
 
-sys.path.append(str(Path(__file__).resolve().parents[4]))
+from syspath_hack import add_to_syspath
+try:
+    from syspath_hack import prepend_project_root
+except ImportError:  # pragma: no cover - compat for older syspath-hack
+    def prepend_project_root(sigil: str = "pyproject.toml") -> Path:
```

Ruff (0.14.6)

Unused function argument: `sigil`

(ARG001)